feat(K8s): Added changes for gather info related to v2 (#17994)
newrelic/docs-website: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```

<Callout variant="tip">

>   Notice we are specifying `--tls --port 10250`. Earlier versions of the integration might have worked without this, as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3) of the Kubernetes integration it is mandatory to specify them. The reason for this change is that the integration now connects to the Kubelet using the nodeIP rather than `localhost`, with the former requiring TLS while the latter did not.

Reviewer: ```suggestion
 Note that we're specifying `--tls --port 10250` in the command. While previous versions of the integration might have functioned without these flags, they are now mandatory as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3). This change is due to the integration's connection method shifting from `localhost` to the nodeIP, which requires TLS encryption.
```

</Callout>

Integrations targeting other pods should have their own section next to `redis-sampleapp.yaml`.

> Integrations are standalone binaries and are executed by the infrastructure agent that's included in the `newrelic-nrk8s-kubelet-xxxxx` pods. Config files are deployed to all pods of the `nrk8s-kubelet` DaemonSet, but discovery ensures that each pod will only target service pods that're scheduled in the same node as that particular `nrk8s-kubelet` pod. If an instance of the `nrk8s-kubelet` DaemonSet does not have any pods matching the specified labels, the integration will not be executed by that ### instance.

Reviewer: ```suggestion
Integrations are standalone binaries and are executed by the infrastructure agent that's included in the `newrelic-nrk8s-kubelet-xxxxx` pods. Config files are deployed to all pods of the `nrk8s-kubelet` DaemonSet, but discovery ensures that each pod will only target service pods that're scheduled in the same node as that particular `nrk8s-kubelet` pod. If an instance of the `nrk8s-kubelet` DaemonSet does not have any pods matching the specified labels, the integration will not be executed by that `###` instance.
```

I'm not sure about what ### is referencing, but if it's something code, put it in code blocks

### Add a service YAML to the Kubernetes integration config [#configmap]

> In order for the snippet above to be recognized by the integration, it must be placed inside the designated ConfigMap. If you're using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), a ConfigMap should already be generated, with a name ending in `-integrations-cfg`. 

Reviewer: ```suggestion
To be recognized by the integration, the code snippet must be placed within a designated `ConfigMap`. If you're using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), a `ConfigMap` with a name ending in `-integrations-cfg` should already exist.
```

Reviewer: Check other references of ConfigMap on this page and make sure they're wrapped in code format

Author: I've rewritten the first sentence to avoid the passive form:
`The integration will recognize the above snippet, if you place it inside a designated `ConfigMap``
I checked the other references of ConfigMap and now they are all in code format! Thank you!!

> For the Kubernetes Integration version 2, see [these instructions](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/k8s-version2/#add-service-yaml).

Reviewer: ```suggestion
If you're using Kubernetes Integration version 2, see [Add service yaml](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/k8s-version2/#add-service-yaml).
```

          METRICS: 1
```

> If you are using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), this ConfigMap will already be mounted in the required container.

Reviewer: ```suggestion
If you're using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), this `ConfigMap` will already be mounted in the required container.
```
